bodhi/server/templates/new_update.html

```diff
@@ -71,7 +71,7 @@ <h5 class="font-weight-bold my-1 ml-2">Builds</h5>
                           </div>
                       </div>
                       % endif
-                      % if update and update.from_tag:
+                      % if update and update.from_tag and not update.release.composed_by_bodhi:
```

Comment:
I have a hard time to understand this change, do this mean that we will only show this button for rawhide ?

Reply:
Yes, it does. Because the update (and its builds) will be moved into the normal workflow, i.e. re-tagged with the pending-signing/-testing tag, and the side-tag will be removed, having the button serves no purpose because the tag doesn't exist anymore at this point.

Reply:
Ok I misunderstood the purpose of this PR. I am not sure I get the use case. How does someone creates an update from a side tag in a stable release then ?

Reply:
> How does someone creates an update from a side tag in a stable release then ?

The user creates the side-tag, builds in it, creates an update from that side-tag.
At this point, all the builds are merged back to the traditional koji tag (the default signing-pending) and the side-tag is removed.
If the user needs to fix something for that update, they will have to go the buildroot override way as they do today.

Reply:
> > How does someone creates an update from a side tag in a stable release then ?
> 
> The user creates the side-tag, builds in it, creates an update from that side-tag.
> At this point, all the builds are merged back to the traditional koji tag (the default signing-pending) and the side-tag is removed.
> If the user needs to fix something for that update, they will have to go the buildroot override way as they do today.

Ok I missed that this was for editing the update and not creating it. Looking at the name of the template I assumed that this was only used to create new updates :see_no_evil:

Reply:
There are some confusingly named templates, methods, etc. around new updates vs. editing. :slightly_frowning_face: